control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -271,6 +274,9 @@ func (r *Reconciler) reconcileBrokerTopic(broker *eventing.Broker, securityOptio
 
 	statusConditionManager.TopicReady(topicName)
 	logger.Debug("Topic created", zap.Any("topic", topicName))
+
+	broker.Status.Annotations[kafka.TopicAnnotation] = topicName
```

Comment:
to read it in the trigger reconciler instead of assuming or duplicating the logic for internal or external topics (or whatever other logic we come up next).

Reply:
If we stash more content on the `status annotation`, you think this is still "OK" ? 

https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/reconciler/broker/broker.go#L503

as the topic name is in theory not on that `kafka-broker-config` CM

Reply:
I think it's ok, it's mostly to decouple concerns, broker owns the topic

 (Btw, I also need this for the scaling feature)